helm/kanister-operator/templates/rbac.yaml

```diff
@@ -24,6 +24,12 @@ rules:
   - customresourcedefinitions
   verbs:
   - "*"
+- apiGroups:
+  - snapshot.storage.k8s.io
+  resources:
+  - "*"
+  verbs:
+  - "*"
```

Comment:
do we need `*`s here.

Reply:
The functions needs  for verbs because we want permission to create, get, delete

Reply:
hmm, what about resources?
there are other verbs as well for example list and patch but I think if we are allowing create and delete we can use `*`.

Reply:
For resources, we are using VolumeSnapshot and VolumeSnapshotContent

Reply:
@PrasadG193 @viveksinghggits Travis CI check is failing when I mention specific resources and verbs in clusterrole template. Error log shows some issue in pkg/app/mongodb.go file. Is this related to issues in e2e tests for mongo app?

Reply:
is it not logging any error messages?

Reply:
Here is the error log

`FAIL: <autogenerated>:1: MongoDB.TestRun
integration_test.go:188:
    c.Assert(err, IsNil)
... value *exec.ExitError = &exec.ExitError{ProcessState:(*os.ProcessState)(0xc000900030), Stderr:[]uint8(nil)} ("signal: killed")
time="2021-12-31T17:38:06.672662447Z" level=info msg="Uninstalling application." File=pkg/app/mongodb.go Function="github.com/kanisterio/kanister/pkg/app.(*MongoDB).Uninstall" Line=134 app=mongo hostname=travis-job-6c03e726-94bb-4c14-8500-7f06ebe99983
OOPS: 5 passed, 1 FAILED
--- FAIL: Test (468.44s)
FAIL
exit status 1
FAIL	github.com/kanisterio/kanister/pkg/testing	468.501s
Makefile:199: recipe for target 'run' failed
make[1]: *** [run] Error 1
make[1]: Leaving directory '/home/travis/build/kanisterio/kanister'
Makefile:163: recipe for target 'integration-test' failed
make: *** [integration-test] Error 2
The command "make integration-test" exited with 2.
before_cache
0.09s$ sudo chown -R travis:travis ./.go
cache.2
store build cache
after_script
4.95s$ make stop-kind
Done. Your build exited with 1.`

Reply:
Like discussed this is not because of our RBAC change, let's rebase this branch with master and this should be resolved.